ui-v2/lib/startup/index.js

```diff
@@ -1,7 +1,36 @@
 /* eslint-env node */
 'use strict';
+//
+const $ = process.env;
+const fs = require('fs');
+const path = require('path');
+const promisify = require('util').promisify;
+const read = promisify(fs.readFile);
+const apiDouble = require('@hashicorp/api-double');
+
+const apiDoubleHeaders = require('@hashicorp/api-double/lib/headers');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+//
 module.exports = {
   name: 'startup',
+  serverMiddleware: function(server) {
+    // TODO: This should all be moved out into ember-cli-api-double
+    // and we should figure out a way to get to the settings here for
+    // so we can set this path name centrally in config
```

Comment:
Here's the docs for that: https://ember-cli.com/api/classes/Addon.html#method_serverMiddleware - maybe something in `server.project.options` ?

Reply:
Hehe yeah I dove down into all sorts, and then decided to call this a wrap. It seemed like all the things/configs/properties were coming direct from the `package.json` files for all the addons. I'm definitely gonna return to this to make it configurable elsewhere though. I figured it was ok to drop in as it is though and get back on to that later. Thanks for the link!